templates/nginx.conf.j2

```diff
@@ -0,0 +1,34 @@
+user  nginx;
+worker_processes  auto;
+worker_rlimit_nofile {{ nginx_rlimit_nofile }};
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+events {
+    worker_connections  {{ nginx_worker_connections }};
+}
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" $status $body_bytes_sent "$http_referer" "$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
```

Comment:
here, it would be `access_log /dev/stdout;`.

Reply:
`nginx[28184]: nginx: [emerg] open() "/dev/stdout" failed (6: No such device or address`

That doesn't work.